src/js/components/DataTable/Header.js

```diff
@@ -183,6 +193,19 @@ const Header = ({
               }
             } else content = header;
 
+            if (unitsContent) {
+              content = (
+                <Box
+                  direction="row"
+                  align={theme.dataTable.header.units.align}
+                  gap={theme.dataTable.header.units.gap}
```

Comment:
I think the three of those should be under `theme.dataTable.header.units.container` and let the container accept any BoxProps. hence will clean up this code to:
```
  <Box {...theme.dataTable.header.units.container}> ... </Box>
```

Reply:
I like it...

Reply:
Done